### ISSUE TITLE
README Wrong Instructions

### ISSUE THREAD

**Issue by LuckyRepo** [libnidsj/patchrom_angler#10]
git clone https://github.com/libnijunior/patchrom_bullhead -b mtc20l angler

should be

git clone https://github.com/libnijunior/patchrom_angler -b mtc20l angler

**Comment by patch-Addams** [comment]
I noticed the same thing when building this.  I should get in a better habit of posting here.

**Comment by LuckyRepo** [comment]
Good idea. Did you get a `make: *** [out/InCallUI.apk] Error 1` when making ota?

**Comment by patch-Addams** [comment]
No sir.  The build built fine for me.
